### PR TITLE
docs: improve helpers

### PR DESCRIPTION
**Pull request by kenjis**
**Description**
From https://forum.codeigniter.com/showthread.php?tid=88591&pid=414184#pid414184

- improve descriptions

**Checklist:**
- [x] Securely signed commits
- [] Component(s) with PHPDoc blocks, only if necessary or adds value
- [] Unit testing, with >80% coverage
- [x] User guide updated
- [x] Conforms to style guide

### CODE REVIEW

**Review by datamweb — APPROVED** [review]
This explanation is good.
But what makes documents simple and fluent is the existence of examples in the document. I think creating a title with "creating custom helper" and a simple example is good for improving the documentation.
```php
<?php
// app/Helpers/info_helper.php
use CodeIgniter\CodeIgniter;

if (!function_exists('getVer')) {

    function getVer()
    {
        return CodeIgniter::CI_VERSION;
    }
}
```

**Review by neznaika0 — APPROVED** [review]
It's clear to me how you can create a helper

File: user_guide_src/source/general/helpers/008.php [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?php
+
+// app/Helpers/info_helper.php
+
+/**
+ * Returns CodeIgniter's version.
+ */
+function ci_version(): string
+{
+    return CodeIgniter::CI_VERSION;
```
Comment (datamweb) [inline user_guide_src/source/general/helpers/008.php:10]:
```suggestion
    return CodeIgniter\CodeIgniter::CI_VERSION;
```
OR use CodeIgniter\CodeIgniter;

Comment (kenjis) [reply user_guide_src/source/general/helpers/008.php:11]: Done.

Comment (datamweb) [reply user_guide_src/source/general/helpers/008.php:11]: Thanks @kenjis.

**Review by michalsn — APPROVED** [review]

**Pull request merged**